### ISSUE TITLE
[BUG] Error loading dcm from directory

### ISSUE THREAD

**Issue by JMZAM** [pyvoxel/pyvoxel#51]
**Describe the bug**
An error is generated when a dcm directory that includes "." in the name is specified as a path.

**To Reproduce**
import voxel as vx
path_to_ct = '/sample/path/to/CT/series/1.2.840/'
ct_scan = vx.load(path_to_ct)

ValueError: Unknown data format for /sample/path/to/CT/series/1.2.840/

**Expected behavior**
Expected voxel to recognize the path as a directory containing dices and load a medical volume.

**Desktop (please complete the following information):**
Linux 
5.15.0-52-generic 
#58-Ubuntu SMP 
x86_64 
GNU/Linux

Please also provide the output of the following code:

```python
>>> from voxel.utils.collect_env import collect_env_info
>>> print(collect_env_info())
```
'------------  -------------------------------------------------------------------------------\nsys.platform  linux\nPython        3.9.13 (main, Oct 13 2022, 21:15:33) [GCC 11.2.0]\nnumpy         1.23.3\nvoxel         0.0.1 @.../lib/python3.9/site-packages/voxel\nnibabel       4.0.2\npydicom       2.3.0\n------------  -------------------------------------------------------------------------------'

**Notes**
This may be because of how you're detecting dcm dirs in `voxel/io/format_io.py:76`. i.e. using os.path.splitext() splits the final directory name on '.'

**Comment by ad12** [comment]
thanks for the issue! could you list out the files underneath that path

**Comment by sluijs** [comment]
@JMZAM I can only reproduce this behavior if I remove the trailing slash `/sample/path/to/CT/series/1.2.840/` -> `/sample/path/to/CT/series/1.2.840`. Can you confirm that?

**Comment by JMZAM** [comment]
@ad12 the directory looks like this:
/sample/path/to/CT/series/1.2.840/
|____ 0001-1.2.840.3275...dcm
|____ 0002-1.2.840.3298...dcm
|____ 0003-1.2.840.3209...dcm
|____ ...

@sluijs adding the trailing / at the end resolves the issue!

may consider os.path.isdir(/path/to/CT/series/) and glob.glob('*.dcm') for your checks on  `voxel/io/format_io.py:76`

**Comment by sluijs** [comment]
Great, should be resolved in #52
